content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -306,11 +308,12 @@ without compromising the minimum required capacity for running your workloads.
 
 ### Drain the node
 
-1.  Prepare the node for maintenance by marking it unschedulable and evicting the workloads. Run:
+1.  Prepare the node for maintenance by marking it unschedulable and evicting the workloads:
 
     ```shell
```

Comment:
I would add a shell comment before the command:
```bash
# Replace `<node-name>` with the name of your node you are draining
```
You can do that @sharjeelaziz, or tweak the wording for the explanatory line.
“the name of your node” might possibly confuse a non-native reader who has more than one node.

Same choice elsewhere in the document.

Reply:
Excellent suggestions. Just to be clear your suggestion is to incorporate the note as a shell comment above the command so that it looks something like this:
   ```shell
   # Replace <node-name> with the name of your node you are draining
   kubectl drain <cp-node-name> --ignore-daemonsets
```
Any other suggestions on how to go about committing changes with your suggestions.

Reply:
No further suggestions.